packages/node_modules/@webex/webex-core/test/integration/spec/services/services.js

```diff
@@ -288,6 +288,18 @@ describe('webex-core', () => {
       });
     });
 
+    describe('#hasService()', () => {
```

Comment:
Since these don't interact with any actual services, should they be unit tests instead of integration?